tests/integ/modin/test_apply_snowflake_cortex_functions.py

```diff
@@ -81,20 +82,54 @@ def test_apply_snowflake_cortex_sentiment_df(session):
     running_on_jenkins(),
     reason="TODO: SNOW-1859087 snowflake.cortex.sentiment SSL error",
 )
-def test_apply_snowflake_cortex_classify_text(session):
+@pytest.mark.parametrize(
+    "is_series, operation, query_count",
+    [
+        param(
+            True,
+            (lambda s: s.apply(ClassifyText, categories=["travel", "cooking"])),
+            1,
+            id="series_cortex_classify_text_kwargs",
+        ),
+        param(
+            False,
+            (lambda df: df.apply(ClassifyText, categories=["travel", "cooking"])),
+            2,
+            id="df_cortex_classify_text_kwargs",
+        ),
+        param(
+            True,
+            (lambda s: s.apply(ClassifyText, args=(["travel", "cooking"]))),
```

Comment:
```suggestion
            (lambda s: s.apply(ClassifyText, args=(["travel", "cooking"],))),
```

`args` without the comma is the list `["travel", "cooking"]` itself rather than a tuple containing that list.